TESTING.md

```diff
@@ -50,7 +50,7 @@ py.test -m "not requires_pufcsv"
 ```
 
 This will start executing a pytest suite containing about 300 tests,
-but will skip the few tests that require the `puf.csv` file as input.
+but will skip the three tests that require the `puf.csv` file as input.
```

Comment:
Should we really specify the exact number of tests in this document? We'll have to constantly update them, and the actual count is easily deducible for someone running `py.test`.